### PR TITLE
Align checkbox and radio buttons with labels on surveys

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

[Align checkbox and radio buttons with labels on surveys](https://github.com/webkom/lego-webapp/commit/51614857dd3c8b3b9c0797074be2e8f4eca3839a)

---
  
[Minor improvements to surveys](https://github.com/webkom/lego-webapp/commit/5a7b6388a746180cf42f5cadcab1a62cdad71b7e) 

Mostly type improvements.

Changed how the copy token button handles clicks. It's now less glitchy
if you spam it, so that's nice, I guess ..

# Result

| Before | After |
:-----------------------:|:--------------------------:
image | image
<video src="https://user-images.githubusercontent.com/69514187/233858102-39da48ea-618c-4eda-b52a-54f31def5717.mov" /> | <video src="https://user-images.githubusercontent.com/69514187/233858099-9ecc12c6-620d-4311-ba6b-1bd1c8f190d2.mov" />

# Testing

- [x] I have thoroughly tested my changes.

Labels are now aligned and copy button is flawless.

### CODE REVIEW

**Review by LudvigHz — APPROVED** [review]

File: app/routes/surveys/utils.tsx
Comment (ingraso) [inline app/routes/surveys/utils.tsx:1]: Is the `cx` import used anymore? Or else, could it be removed?

**Review by ingraso — APPROVED** [review]
Great work! Much better code here now. Just added a minor comment :smile:

File: app/routes/surveys/utils.tsx
Comment (ivarnakken) [reply app/routes/surveys/utils.tsx:1]: Yes, it is still used on line 112 and 137 😄

Comment (ingraso) [reply app/routes/surveys/utils.tsx:1]: Oh okay, my bad! :sweat_smile:

**Pull request merged**